schema/home/home_page_hero_units.js

```diff
@@ -21,10 +22,34 @@ const HomePageHeroUnitType = new GraphQLObjectType({
       type: GraphQLString,
       resolve: ({ link }) => link,
     },
+    description: {
+      type: GraphQLString
+    },
     title: {
       type: GraphQLString,
       resolve: ({ mobile_title }) => mobile_title,
```

Comment:
We'll probably want to make MP smarter about what it's returning for `title` now based on the `platform` below?